features/support/env.rb

```diff
@@ -23,6 +23,14 @@
   Delayed::Worker.delay_jobs = false
 end
 
+Before('@freeze_time') do
+  Timecop.freeze(5.weeks.ago)
```

Comment:
I just realized how foolish this is. It will cause the errors to return in `5.weeks`. I will update.